src/controllers/requestController.js

```diff
@@ -53,31 +53,34 @@ export default class requestController {
   // creating a request
   static async createRequest(req, res) {
     try {
-      let _response = '';
-      const newIdRoom = req.body.idRoom,
-        allRequests = await findRequestByRoomId(newIdRoom);
+      const { idRoom, hotelName, dateStart, dateEnd } = req.body,
```

Comment:
would you consider changing starting date and ending date to date format as you get if as string from user